packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/FLTCaptureVideoDataOutput.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+@import AVFoundation;
+
+#import "FLTCaptureOutput.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// A protocol which is a direct passthrough to `AVCaptureVideoDataOutput`. It exists to allow
+/// mocking `AVCaptureVideoDataOutput` in tests.
+@protocol FLTCaptureVideoDataOutput <FLTCaptureOutput>
+
+/// The underlying instance of `AVCaptureVideoDataOutput`.
+@property(nonatomic, readonly) AVCaptureVideoDataOutput *avOutput;
+
+@property(nonatomic) BOOL alwaysDiscardsLateVideoFrames;
```

Comment:
these all need doc comments

Reply:
What kind of comment do you have in mind? More "Corresponds to the alwaysDiscardsLateVideoFrames property of the wrapped AVCaptureVideoDataOutput" or a copy of the comment from the AVCaptureVideoDataOutput so that docs are quick to access despite the additional wrapper?

Reply:
Either seems reasonable.

Reply:
I added the former. IMO those comments are mostly pointless. What else can a wrapper method do besides calling the equivalent in the wrapped instance? The latter would make more sense but I'm not sure how copyright/licensing around apple documentation works in this case so I don't want to copy it

Reply:
> IMO those comments are mostly pointless.

FWIW, I agree. I'm a stickler for enforcing the style guide's requirements for declaration comments in my reviews, but for protocols/classes that are documented at the class level explicitly as being direct passthroughs I have been making an exception since in that specific context I don't think they are adding anything.

> The latter would make more sense but I'm not sure how copyright/licensing around apple documentation works in this case so I don't want to copy it

If we are going to put comments, maybe they should just be links to the actual Apple docs for the method.